File: test/fixtures/stdlib/string/length.expected.py
(lambda _: kAssert(len("hello") == 5))(None)
(lambda _: kAssert(len("") == 0))(None)
(lambda _: kAssert(len("abc def") == 7))(None)
(lambda _: kAssert(len("ab" + "cd") == 4))(None)

File: test/fixtures/stdlib/string/summary/length.expected.js
(function(_) { return (function() { if (!("hello".length == 5)) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("".length == 0)) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!("abc def".length == 7)) throw new Error("Assertion failed"); return true; })(); })(null);
(function(_) { return (function() { if (!(("ab" + "cd").length == 4)) throw new Error("Assertion failed"); return true; })(); })(null);
